Description: IBM Websphere is non-free (remove it from source)
Author: Damien Raude-Morvan <drazzib@debian.org>
Last-Update: 2010-06-20
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -578,6 +578,15 @@
 		testCompile("org.eclipse.persistence:javax.persistence:2.0.0")
 		testCompile("org.codehaus.groovy:groovy-all:${groovyVersion}")
 	}
+
+	sourceSets {
+		main {
+			java {
+				srcDir 'src/java'
+				exclude '**/WebSphereUowTransactionManager.java'
+			}
+		}
+	}
 }
 
 project("spring-jms") {
